datalad/distribution/create_sibling.py

```diff
@@ -92,7 +92,7 @@ def _create_dataset_sibling(
         ds_target_pushurl = target_pushurl.replace('%RELNAME', ds_name) \
             if target_pushurl else ds_sshurl
 
-    lgr.info("Creating target dataset {0} at {1}".format(
+    lgr.info("Considering to create a target dataset {0} at {1}".format(
```

Comment:
Who said that we are honest here?  but we are indeed considering (or analyzing or ...), but not yet creating really